internal/exec/resolvable/resolvable.go

```diff
@@ -173,7 +173,7 @@ func (b *execBuilder) makeExec(t common.Type, resolverType reflect.Type) (Resolv
 		return e, nil
 
 	default:
-		panic("invalid type")
+		panic("invalid type: " + t.String())
```

Comment:
I'm interested to see cases which cause this panic.

Reply:
just make a typo: define input type as `type`, not `input` :)